Makefile

```diff
@@ -47,6 +48,7 @@ ALL_BUILDABLE_FLAVORS := \
 	luminous,centos,7 \
 	kraken,centos,7 \
 	luminous,opensuse,42.3 \
+	luminous,debian,stretch-backports \
```

Comment:
ditto

Reply:
Hum that makes me think that at some place you have some _9 values too. So maybe you could have 9 here, having the __OS_VERSION__ equals stretch and having some code adding __OS_VERSION__-backports repo.

Reply:
Agree